crates/autopilot/src/infra/persistence/dto/fee_policy.rs

```diff
@@ -48,3 +48,31 @@ pub fn from_domain(
         },
     }
 }
+
+pub fn into_domain(
+    policy: FeePolicy,
+    quote: &domain::quote::Quote,
```

Comment:
Probably a bigger refactor awaiting but it seems odd to me that we need to have a domain object of a quote to turn the infra FeePolicy into a domain Policy.
Shouldn't the `persistence` layer be capable of recovering the required quote data?

Reply:
It should, however we don't have it implemented yet. Instead, I reused what we currently have (similar to what we did [here](https://github.com/cowprotocol/services/blob/main/crates/autopilot/src/database/auction.rs#L76-L87)). If you think this should be part of the colocation feature and launch blocking, I will create an issue for it.